src/diagrams/er/erDb.js

```diff
@@ -69,15 +72,17 @@ const getRelationships = () => relationships;
 
 // Keep this - TODO: revisit...allow the diagram to have a title
 const setTitle = function (txt) {
-  title = txt;
+  let sanitizedText = sanitizeText(txt, configApi.getConfig());
```

Comment:
Do we need to pass in `configApi.getConfig()` here since it's already passed into the `sanitizeText` function on line 11?

Reply:
You are right. That is not necessary. We can remove that.